src/molecules/formfields/SelectField/index.js

```diff
@@ -31,10 +31,11 @@ function SelectField ( props ) {
       }
 
       <div className={ styles.selectbox }>
-        <select required
+        <select
+          required
           className={ styles.select }
           onChange={ onChange }
-          value={ value }
```

Comment:
how come you switched this out in favor of `defaultValue`. it was my understanding that `defaultValue` is for uncontrolled components?